v20.2/drop-column.md

```diff
@@ -22,7 +26,7 @@ The user must have the `CREATE` [privilege](authorization.html#assign-privileges
 -----------|-------------
  `table_name` | The name of the table with the column you want to drop.
  `name` | The name of the column you want to drop.<br><br>When a column with a `CHECK` constraint is dropped, the `CHECK` constraint is also dropped.
- `CASCADE` | Drop the column even if objects (such as [views](views.html)) depend on it; drop the dependent objects, as well.<br><br>`CASCADE` does not list objects it drops, so should be used cautiously. However, `CASCADE` will not drop dependent indexes; you must use [`DROP INDEX`](drop-index.html).<br><br>`CASCADE` will drop a column with a foreign key constraint if it is the only column in the reference.
+ `CASCADE` | Drop the column even if objects (such as [views](views.html)) depend on it; drop the dependent objects, as well. `CASCADE` will drop a column with a foreign key constraint if it is the only column in the reference.<br><br>`CASCADE` does not list the objects it drops, so should be used cautiously.<br><br><span class="version-tag">New in v20.2:</span> `CASCADE` is not required to drop an indexed column. `DROP COLUMN` drops the column and any indexes on the column by default.
```

Comment:
[nit] `DROP COLUMN` will also drop a partial index with a predicate that references the column being dropped. For example, `ALTER TABLE t DROP COLUMN c` would drop an `INDEX ON t (b) WHERE c > 0`.

Reply:
`ALTER TABLE t DROP COLUMN c` would also drop an `INDEX ON t (b) STORING (c)`.